### ISSUE TITLE
Not rendering current time and duration correctly after order change in customProgressBarSection

### ISSUE THREAD

**Issue by isenmann** [lhz516/react-h5-audio-player#94]
**Describe the bug**

In my project I need to switch the customProgressBarSection depending on the track which is played. I have found out that by doing this the RHAP_UI.CURRENT_TIME and RHAP_UI.DURATION won't be rendered anymore correctly (renders only --:--) in the progress bar section. If you want to reproduce this then just change the order of the customProgressBarSection like shown in the example between each track or between setting a new src for another track.
The progress bar is working, no matter on which position it's set during the switch. 

```
var customProgressBarSection = this.state.switchTrack ? [RHAP_UI.CURRENT_TIME, RHAP_UI.PROGRESS_BAR, RHAP_UI.DURATION] : [RHAP_UI.DURATION, RHAP_UI.PROGRESS_BAR, RHAP_UI.CURRENT_TIME];

<AudioPlayer style={{backgroundColor: "transparent"}} 
 layout="horizontal"
 src={playTrackSrc}
 onEnded={e => this.nextTrack()}
 onClickNext={e => this.nextTrack()}
 onClickPrevious={e => this.previousTrack()}
 showSkipControls={true}
 customProgressBarSection={customProgressBarSection} />
```

**Environment**
Package version: 3.5.0
React version: 16.0.0
Browser and its version: Chrome  88.0.4324.150
OS and its version: Windows 10 (64bit) version 20H2

**Comment by lhz516** [comment]
Thanks for reporting. Fixed and published 3.6.1
Please try it out

**Comment by isenmann** [comment]
Amazing! This was a very fast fix. Works like a charm...
